src/js/index.d.ts

```diff
@@ -0,0 +1,200 @@
+export * as constants from "./constants.js";
```

Comment:
NOTE: Going to drop this in this PR, but this was the quick fix solution to get all of the TypeScript types in place for React. We should instead just completely revamp how the React wrappers work in our v1.0 effort.